docs/security.rst

```diff
@@ -72,9 +72,9 @@ New releases for OpenSSL updates
 --------------------------------
 
 As of versions 0.5, 1.0.1, and 2.0.0, ``cryptography`` statically links OpenSSL
-on Windows, macOS, and Linux respectively, to ease installation. Due to this,
-``cryptography`` will release a new version whenever OpenSSL has a security or
-bug fix release to avoid shipping insecure software.
+in binary distributions for Windows, macOS, and Linux respectively, to ease
+installation. Due to this, ``cryptography`` will release a new version whenever
+OpenSSL has a security or bug fix release to avoid shipping insecure software.
```

Comment:
This just adds "in binary distributions for" (replacing "in" with "for"), but also rewraps at 79 characters to match existing style (though it makes it harder to see the changes, figuring the consistent wrapping was preferred here, but if not I'm happy to not rewrap).